docs/dependency-updates.md

```diff
@@ -0,0 +1,67 @@
+# Dependency Updates
+
+Each week we rotate a few of the routine process and maintenance chores between team members. This is tracked using a schedule posted to the USTC out of office calendar. 
+
+At the moment, the only task we rotate is updating dependencies. As an open-source project, we rely on external libraries which get updates frequently. These include JavaScript Library Updates and Infrastructure as Code Libraries as described below. Follow the [PR workflow](./pr-workflow.md) like any other change.
+
+> After changes are made to any dependencies, deploy to an experimental environment to verify that all tests pass!
+
+## Library Update Steps
+
+1. `npm update`: Update to current minor versions of all libraries. These shouldn't include any breaking changes, but still might, so it's best to verify with smoke tests in AWS.
```

Comment:
All of these numbered points are 1?

Reply:
Yeah, good observation! That was intentional - James addressed that [here](https://github.com/ustaxcourt/ef-cms/pull/2617#discussion_r988326741)